django_jsonapi/settings/base.py

```diff
@@ -151,8 +151,8 @@
     'DEFAULT_PAGINATION_CLASS':
         'rest_framework_json_api.pagination.JsonApiPageNumberPagination',
     'DEFAULT_PARSER_CLASSES': (
-        'rest_framework.parsers.JSONParser',
-        # 'rest_framework_json_api.parsers.JSONParser',
+        # 'rest_framework.parsers.JSONParser',
```

Comment:
remove the comment/unused code, please.

Reply:
This is a temporary fix, would require looking further into the parser.